<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Picsou</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../picsou.qrc">
    <normaloff>:/resources/picsou.png</normaloff>:/resources/picsou.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tab_widget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="details_tab">
       <attribute name="icon">
        <iconset resource="../picsou.qrc">
         <normaloff>:/resources/material-design/svg/information-outline.svg</normaloff>:/resources/material-design/svg/information-outline.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Details</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="search_tab">
       <attribute name="icon">
        <iconset resource="../picsou.qrc">
         <normaloff>:/resources/material-design/svg/magnify.svg</normaloff>:/resources/material-design/svg/magnify.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
      <widget class="QWidget" name="graph_tab">
       <attribute name="icon">
        <iconset resource="../picsou.qrc">
         <normaloff>:/resources/material-design/svg/finance.svg</normaloff>:/resources/material-design/svg/finance.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Charts</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Charts coming soon* !&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>*or not, depends on developper free time...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1143</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_close"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about_picsou"/>
    <addaction name="action_about_qt"/>
    <addaction name="separator"/>
    <addaction name="action_report_an_issue"/>
    <addaction name="action_star_me_on_github"/>
    <addaction name="action_license"/>
   </widget>
   <widget class="QMenu" name="settings_menu">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="action_preferences"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="settings_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="action_close"/>
   <addaction name="separator"/>
   <addaction name="action_save"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="tree_dock">
   <property name="windowIcon">
    <iconset resource="../picsou.qrc">
     <normaloff>:/resources/material-design/svg/file-tree.svg</normaloff>:/resources/material-design/svg/file-tree.svg</iconset>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Database tree</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeWidget" name="tree">
       <property name="animated">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_open">
   <property name="icon">
    <iconset resource="../picsou.qrc">
     <normaloff>:/resources/material-design/svg/folder-open.svg</normaloff>:/resources/material-design/svg/folder-open.svg</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="../picsou.qrc">
     <normaloff>:/resources/material-design/svg/exit-to-app.svg</normaloff>:/resources/material-design/svg/exit-to-app.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset resource="../picsou.qrc">
     <normaloff>:/resources/material-design/svg/folder-remove.svg</normaloff>:/resources/material-design/svg/folder-remove.svg</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_about_picsou">
   <property name="icon">
    <iconset resource="../picsou.qrc">
     <normaloff>:/resources/material-design/svg/currency-eur.svg</normaloff>:/resources/material-design/svg/currency-eur.svg</iconset>
   </property>
   <property name="text">
    <string>About Picsou...</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="icon">
    <iconset resource="../picsou.qrc">
     <normaloff>:/resources/qt_extended_48x48.png</normaloff>:/resources/qt_extended_48x48.png</iconset>
   </property>
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="action_star_me_on_github">
   <property name="icon">
    <iconset resource="../picsou.qrc">
     <normaloff>:/resources/material-design/svg/github-box.svg</normaloff>:/resources/material-design/svg/github-box.svg</iconset>
   </property>
   <property name="text">
    <string>Star me on Github...</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="../picsou.qrc">
     <normaloff>:/resources/material-design/svg/content-save.svg</normaloff>:/resources/material-design/svg/content-save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_new">
   <property name="icon">
    <iconset resource="../picsou.qrc">
     <normaloff>:/resources/material-design/svg/folder-plus.svg</normaloff>:/resources/material-design/svg/folder-plus.svg</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_license">
   <property name="icon">
    <iconset resource="../picsou.qrc">
     <normaloff>:/resources/material-design/svg/book-open.svg</normaloff>:/resources/material-design/svg/book-open.svg</iconset>
   </property>
   <property name="text">
    <string>License information...</string>
   </property>
  </action>
  <action name="action_report_an_issue">
   <property name="icon">
    <iconset resource="../picsou.qrc">
     <normaloff>:/resources/material-design/svg/bug.svg</normaloff>:/resources/material-design/svg/bug.svg</iconset>
   </property>
   <property name="text">
    <string>Report an issue...</string>
   </property>
  </action>
  <action name="action_preferences">
   <property name="icon">
    <iconset resource="../picsou.qrc">
     <normaloff>:/resources/material-design/svg/settings.svg</normaloff>:/resources/material-design/svg/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../picsou.qrc"/>
 </resources>
 <connections/>
</ui>
